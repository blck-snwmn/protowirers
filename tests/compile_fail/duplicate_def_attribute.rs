use protowirers::*;

#[derive(Proto)]
struct Sample {
    #[def(field_num = 1, def_type = "int32")]
    #[def(field_num = 1, def_type = "int32")]
    s: u32,
}

fn main() {}
