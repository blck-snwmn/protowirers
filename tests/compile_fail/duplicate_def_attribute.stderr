error: only one #[def(...)] attribute is allowed
 --> $DIR/duplicate_def_attribute.rs:5:5
  |
5 | /     #[def(field_num = 1, def_type = "int32")]
6 | |     #[def(field_num = 1, def_type = "int32")]
7 | |     s: u32,
  | |__________^
